gui/web/src/kelp-ops-api/sendMetricEvent.js

```diff
@@ -0,0 +1,11 @@
+export default (baseUrl, name, eventData) => {
+    return fetch(baseUrl + "/api/v1/sendMetricEvent", {
+        method: "POST",
+        body: {
+            name: name,
```

Comment:
format is good, but shouldn't this be `event_name` and `event_data` as the fields?

this should have been caught in testing before submitting as a PR. it's always best practice to test your code before submitting PRs so we can avoid raising such issues

Reply:
Thanks for catching, works now - I just missed a manual test with all the back and forth.